### ISSUE TITLE
How should I use this from clients (browser)?

### ISSUE THREAD

**Issue by changbowen** [vanillaes/csv#19]
`<script src="path to js file" type="module"></script>` doesn't seem to do anything...

**Comment by evanplaice** [comment]
Oh, right. I need to update the `README.md` to mention this

```javascript
<script type="module">
import { parse } from '@vanillaes/csv';
// then you can call parse() in your code 
</script>
```

**Comment by evanplaice** [comment]
Fixed in [v2.1.15](https://github.com/vanillaes/csv/releases/tag/v2.1.15)
